packages/tangle/timemanager.go

```diff
@@ -4,15 +4,15 @@ import (
 	"sync"
 	"time"
 
+	"github.com/cockroachdb/errors"
```

Comment:
I don't think this is formatted according to `goimports -local github.com/iotaledger/goshimmer`, so I am wondering why the Dog is not complaining :thinking:

Reply:
Executing `goimports` against the file doesn't yield any changes. 🤷🏻

Reply:
Even when doing `goimports -local github.com/iotaledger/goshimmer`? it should group all the goshimmer imports...
But who am I to complain, if the Dog isn't :shrug: